From e89577d30c278fba0f9e40266ceb6fb2821907d5 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Wed, 13 May 2015 08:01:34 +0200
Subject: [PATCH] libzmq: Fix pkg-config files for static linking

Libzmq uses C++ standard library features, so users of it should link
against that as well when statically linking.

Add it to Libs.private so users using pkg-config automatically gets the
correct linker flags.

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 src/libzmq.pc.in       | 1 +
 1 file changed, 1 insertions(+)

diff --git a/src/libzmq.pc.in b/src/libzmq.pc.in
index ba155a3..52a39f7 100644
--- a/src/libzmq.pc.in
+++ b/src/libzmq.pc.in
@@ -7,4 +7,5 @@ Name: libzmq
 Description: 0MQ c++ library
 Version: @VERSION@
 Libs: -L${libdir} -lzmq
+Libs.private: -lstdc++
 Cflags: -I${includedir}
-- 
2.1.4

